packages/grafana-e2e-selectors/src/selectors/components.ts

```diff
@@ -137,8 +137,16 @@ export const Components = {
     active: () => '[class*="-activeTabStyle"]',
   },
   RefreshPicker: {
+    /**
+     * @deprecated use runButtonV2 instead
+     */
     runButton: 'RefreshPicker run button',
+    /**
+     * @deprecated use intervalButtonV2 instead
+     */
```

Comment:
Seeing as these are for the purpose of versioning, maybe change the message to be like "use intervalButtonV2 from Grafana 8.3 instead", or something like that? @hugohaggmark 

And/or maybe the _new_ keys could include the _Grafana_ version instead? dunno. Just a random musing.

Reply:
> use intervalButtonV2 from Grafana 8.3 instead

I love this 😍

Reply:
Sounds good!